index.js

```diff
@@ -125,9 +136,9 @@ const TabGroupPrivate = {
   },
 
   initVisibility: function () {
-    function toggleTabsVisibility(tab, tabGroup) {
```

Comment:
Can you please explain those changes? Most of the code is currently written in pre-ES6 syntax. I agree this should be updated some day, but for now I would prefer to keep the whole code consistent.

Reply:
Yeah that sounds good, sorry I got a bit carried away. I'll rebase this part out. Thanks!

Reply:
@brrd - Okay, function change is reversed. I kept the let values in here because the uses of var differed from a few points. If you want I can rebase those out as well. I'll try to keep this to just a clean commit. Thanks for being awesomely responsive and I hope you're having a good weekend!

Reply:
Thank you for this change. I'm sorry but I don't understand why we should use `let` here. Those variables are not reassigned later. I probably missed something here.

Thank you also for rebasing, this is perfect. However you can totally commit changes back if you feel lazy, I don't really care if the git history is (a bit) messy.

Reply:
@brrd - Sorry for the really late reply. The only reason I used let here was that it's consistent with the other syntax, like [here](https://github.com/brrd/electron-tabs/pull/120/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R68) and [here](https://github.com/brrd/electron-tabs/pull/120/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R88) and [here](https://github.com/brrd/electron-tabs/pull/120/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R207). I didn't want to add the first instance of const to the file or anything, and I'd rather just try and be consistent with the rest of the code. That's all I was really thinking.

Reply:
@toddtarsi Thank you for the explanation. I think we should not use `let` here, it can be confusing in this situation. Please keep `var`, and we will convert then to `const` later, when we will move the whole code base to ES6.

Reply:
@brrd - Okay, let has been reset to var. Let me know if I should make any changes to anything else test or build wise. Thanks!